src/PKSim.Core/Services/SimulationConfigurationTask.cs

```diff
@@ -71,7 +72,7 @@ IApplicationConfiguration applicationConfiguration
 
       public SimulationConfiguration CreateFor(Simulation simulation, bool shouldValidate, bool createAgingDataInSimulation)
       {
-         var module = _objectBaseFactory.Create<Module>().WithName(simulation.Name);
+         var module = _objectBaseFactory.Create<Module>().WithName(simulation.Name).WithIcon(ApplicationIcons.Module.IconName);
```

Comment:
I think the icon (name) should be set in the constructor. Do we do the same for all other building blocks?

Reply:
indeed